src/backend/database_models/agent.py

```diff
@@ -23,11 +23,11 @@ class Model(StrEnum):
 class Agent(Base):
     __tablename__ = "agents"
 
-    version: Mapped[int] = mapped_column(Integer, default=1)
+    version: Mapped[int] = mapped_column(Integer, default=1, nullable=True)
```

Comment:
In which case would this be nullable?

Reply:
i think i misunderstood initially. i changed it so that none of these columns should take in nullable values and if nothing is passed in they should default to the default values here.

the only required values should be `userid`, `model`, `deployment`, `name`. which is reflected by a lack of default value and ive written the unit tests in this PR to check these missing cases.